future-apps/payroll/app/src/screens/components/EmployeeList.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import styled from 'styled-components'
 import BN from 'bn.js'
-import { differenceInYears } from 'date-fns'
+import { totalPaidThisYear } from '../../utils/calculations'
```

Comment:
This could be changed to `utils/calculations` based on https://parceljs.org/module_resolution.html